src/components/Globalstats.js

```diff
@@ -1,32 +1,40 @@
-/* eslint-disable */
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
+import { FaArrowRight } from 'react-icons/fa';
+import { Spinner } from 'react-bootstrap';
 import { fetchCovidData } from '../redux/statisticsSlice';
-import '../App.css'
-import { showDetails } from '../redux/statisticsSlice';
+import CountryStats from './CountryStats';
+import '../App.css';
 
 const Globalstats = () => {
-  const { dataArray/*, isLoading */ } = useSelector((state) => state.Covid);
-    const dispatch = useDispatch();
-  
-  const handleClick = (country) => {
-    dispatch(showDetails(country))
-  }  
-    if (dataArray.length === 0) {
-        setTimeout(() => {
-          dispatch(fetchCovidData());
-        }, '1000');
-      }
-    
+//  const [query, setQuery] = useState('');
+  // const Search = (event) => {
+  //   setQuery(event.target.value);
+  // };
+  const { dataArray, isLoading } = useSelector((state) => state.Covid);
+
+  const filteredData = dataArray.filter((data) => (
+    data.country.toLowerCase()
+    /* .includes(query.toLowerCase()) */
+  ));
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchCovidData());
+  }, [dispatch]);
+
+  if (isLoading) {
+    return <Spinner animation="border" />;
+  }
+
   return (
-    <div className='countryContainer'>
-        {dataArray.map((data) => (
-            <div key={data.country+data.region} className="countryCard" onClick={() => handleClick(data.country+data.region)}>
-                <h2>{data.country}</h2>
-                <h3>{data.region}</h3>
-            </div>
-        ))
-}
-    </div>
-  )}
+    <ul className="itemList">
+      {filteredData.map((data) => (
+        <CountryStats key={data.id} details={data} />
```

Comment:
- [ ] I highly recomment to make the color of the text white like the given design since it white there and it is important to follow the design guidelines 🙏🏻🤩💯

![image](https://github.com/dazehinn/world-covid-19-data/assets/58553711/3382d1a6-f4bf-4591-a8d8-dd8995eb244e)